### PR TITLE
DEVTOOLS-68: Track directories in workspace

### PR DESCRIPTION
**Pull request by juancoen**

### CODE REVIEW

File: tests/unit/pullCommand.test.js [hunk 1]
```diff
@@ -16,12 +16,15 @@ var validateSetupDoneStrategyStub = {};
...
+  files: [
+    {
+      path: '/api.raml',
+      hash: 'asd1231das'
+    }
+  ],
+  directories: []
+};
 
 describe('cleanupCommand', function () {
```
Comment (fschargorosdky) [inline tests/unit/pullCommand.test.js]: Should say 'pullCommand'
It's not your error, but could you fix it here?

File: tests/unit/pullController.test.js [hunk 2]
```diff
@@ -38,7 +42,8 @@ describe('pullController', function () {
   describe('getAPIFiles', run(function (pullController) {
     it('should run correctly', function (done) {
       pullController.getAPIFiles()
-        .then(function (workspaceFiles) {
+        .then(function (result) {
+          console.log(result);
```
Comment (fschargorosdky) [inline tests/unit/pullController.test.js]: forever alone console

File: tests/unit/pullCommand.test.js (hunk 1 above)
Comment (juancoen) [inline tests/unit/pullCommand.test.js]: done

File: tests/unit/pullController.test.js (hunk 2 above)
Comment (juancoen) [inline tests/unit/pullController.test.js]: done

**Comment by fschargorosdky** [comment]
Looks good to me

**Pull request merged**